scripts/updateContributors.mjs

```diff
@@ -0,0 +1,79 @@
+import simpleGit from 'simple-git';
+import { Octokit } from '@octokit/rest';
+import fs from 'node:fs';
+import path from 'path';
+import pMemoize from 'p-memoize';
+import LOST_CONTRIBUTORS from './lost-contributors.json' assert { type: 'json' };
+
+const IGNORED_COMMITS = ['433bbae4f1d4abb50a26306d6679a38ace5c8b78'];
```

Comment:
We should probably extract this to a separate file.

Reply:
If it's just a couple, I don't think it's bad.

Reply:
Yes, not that bad, but we could extend it automatically based on PR tags for example (marking certain PRs as "non-contributions").